### PR TITLE
Documentation

### PR DESCRIPTION
**Pull request by pnrobinson**
added interpretation and cleaned up many other pages

### CODE REVIEW

File: docs/conf.py [hunk 1]
```diff
@@ -46,17 +46,17 @@
...
 # The version info for the project you're documenting, acts as replacement for
 # |version| and |release|, also used in various other places throughout the
 # built documents.
 #
 # The short X.Y version.
-version = u'0.1'
+version = u'1.0.0RC'
 # The full version, including alpha/beta/rc tags.
-release = u'1'
+release = u'1.0.0RC'
```
Comment (julesjacobsen) [inline docs/conf.py]: should't this be '1.0.0-RC1' ?

File: docs/interpretation.rst [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+
+
+.. _rstdiagnosis:
+
+Diagnosis
+~~~~~~~~~
+
+The diagnosis element is meant to refer to the disease that is infered to be present in the individual
+or family being analyzed. The diagnosis can be made by  means of an analysis of the phenotypic or the genomic findings or both.
+The element is optional because if the **resolution_status** is **UNSOLVED** then there is not diagnosis.
```
Comment (julesjacobsen) [inline docs/interpretation.rst]: 'not' -> 'no'

File: docs/protobuf.rst [hunk 3]
```diff
@@ -2,22 +2,28 @@
...
-phenopackets-schema.
+Phenopackets schema uses protobuf, an exchange format developed in 2008 by Google. We refer readers to the
+excellent `Wikipedia page on Protobuf <https://en.wikipedia.org/wiki/Protocol_Buffers>`_ and
+to `Google's documentation <https://developers.google.com/protocol-buffers/>`_ for details. This page
+intends to get curious readers who are unfamiliar with protobuf up to speed with the main aspects of this
+technology, but it is not necessary to understand protobuf to use the phenopacket schema.
+
+Google initially developed Protocol Buffers (protobuf) for internal use, but now has provided a code generator for multiple languages under an open source license. In this documentation, we will demonstrate use of phenopackets-schema with Java, but all of the features are available in any of the languages that protobuf works with including C++ and Python. phenopackets-schema uses protobuf version 3.7.0.
```
Comment (julesjacobsen) [inline docs/protobuf.rst]: 3.7.0-> 3.8.0

File: docs/protobuf.rst [hunk 4]
```diff
@@ -2,22 +2,28 @@
...
 
 
 ~~~~~~~~~~~~~~~~~~~
 Installing protobuf
 ~~~~~~~~~~~~~~~~~~~
 
-The following exercise is not necessary to use phenopackets-schema, but is intended to build intuition for how protobuf works. We first need to install protobuf. We show one simple way of installing protobuf on a linux system in the following.
+The following exercise is not necessary to use phenopackets-schema,
+but is intended to build intuition for how protobuf works.
+We first need to install protobuf. We show one simple way of installing protobuf on a linux system in the following.
```
Comment (julesjacobsen) [inline docs/protobuf.rst]: This isn't true - the `os-maven-plugin` and `protobuf-maven-plugin` will download the correct binary for the system at hand and install it for use in the `target/protoc-plugins` directory.

**Comment by pnrobinson** [comment]
This page was meant as a stand-alone tutorial to see how protoc transforms protobuf into code. The exercise works without maven. I do not find the online documentation for protobuf to be that enlightening. However, perhaps we do not need this here?

**Comment by pnrobinson** [comment]
I added this explanation
The following exercise is not necessary to use phenopackets-schema,
but is intended to build intuition for how protobuf works.
We first need to install protobuf (Note that these intructions are for this tutorial only. The maven system will automatically
pull in protobuf for phenopackets schema).

**Comment by julesjacobsen** [comment]
Ah, sorry - I should have read the entire file rather than just looked at the diff. I see what you were taking about now.

File: docs/interpretation.rst (hunk 2 above)
Comment (pnrobinson) [reply docs/interpretation.rst]: fixed

File: docs/conf.py (hunk 1 above)
Comment (pnrobinson) [reply docs/conf.py]: yes --fixed

File: docs/protobuf.rst (hunk 3 above)
Comment (pnrobinson) [reply docs/protobuf.rst]: fixed

**Comment by pnrobinson** [comment]
OK, I fixed the typos etc.

**Pull request merged**